gradle-plugins/compose/src/test/kotlin/org/jetbrains/compose/test/tests/integration/KotlinCompatibilityTest.kt

```diff
@@ -23,6 +23,22 @@ class KotlinCompatibilityTest : GradlePluginTestBase() {
     @Test
     fun testKotlinJsMpp_1_7_20() = testJsMpp("1.7.20")
 
+    @Suppress("RedundantUnitExpression")
+    @Test
+    fun testKotlinJs_shows_warning_for_androidx_compose_compiler() = testProject(
```

Comment:
Thanks! Can we add a test for jvm and check that it doesn't contain this warning?

Reply:
done